docroot/sites/its.uiowa.edu/modules/its_core/its_core.module

```diff
@@ -107,6 +116,23 @@ function its_core_entity_presave(EntityInterface $entity) {
   }
 }
 
+/**
+ * Implements hook_field_widget_complete_WIDGET_TYPE_form_alter().
+ */
+function its_core_field_widget_complete_paragraphs_form_alter(&$element, &$form_state, $context) {
+  if ($element['widget']['#field_name'] === 'field_alert_updates') {
+    // Don't allow the editor to rearrange the updates,
+    // because they will be sorted chronologically
+    // by the node presave.
+    // Disable the draghandles by disallowing the reference changes,
+    // and also remove the dragdrop mode from the dropdown actions.
+    $element['widget']['#allow_reference_changes'] = FALSE;
+    if (isset($element['widget']['header_actions']['dropdown_actions']['dragdrop_mode'])) {
+      unset($element['widget']['header_actions']['dropdown_actions']['dragdrop_mode']);
+    }
+  }
+}
+
```

Comment:
Would love another set of eyes/thoughts on this. It _works_ but is a bit hacky. It's based on https://www.drupal.org/project/paragraphs/issues/3036020#comment-12989072.

In the future (like if that issue moves forward), there may be an actual setting to disallow dragging instead.

Reply:
Thanks! I added a `@todo` comment to point to the issue for possibly removing later assuming testing doesn't reveal any reason to avoid this.